src/main.rs

```diff
@@ -12,6 +12,12 @@ use policy_server::PolicyServer;
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    // Starting from rustls 0.22, each application must set its default crypto provider.
+    let crypto_provider = rustls::crypto::aws_lc_rs::default_provider();
```

Comment:
```suggestion
    let crypto_provider = rustls::ring::aws_lc_rs::default_provider();
```

`aws_lc_rs` needs additional dependencies on Windows (not relevant here), yet AFAIK we were going with ring everywhere.